pkg/jsonnet/native/funcs.go

```diff
@@ -59,11 +59,7 @@ var parseYAML = &jsonnet.NativeFunction{
 				}
 				return nil, err
 			}
-			jsonDoc, err := json.Marshal(doc)
-			if err != nil {
-				return nil, err
-			}
-			ret = append(ret, jsonDoc)
+			ret = append(ret, doc)
```

Comment:
Wouldn't this result in problems further down the line? Jsonnet expects the result of a native function to comply with the standard Go representation, that is:

```
bool, for JSON booleans
float64, for JSON numbers
string, for JSON strings
[]interface{}, for JSON arrays
map[string]interface{}, for JSON objects
nil for JSON null
```
(source: https://golang.org/pkg/encoding/json/#Unmarshal)

Reply:
yeah we alreaday noticed this